extras/vfd16lf01/LICENSE

```diff
@@ -0,0 +1,7 @@
+This is a driver for Samsung 16LF01 series VFD.
+
+Originate written by Daniele Napolitano, distributed under GPL.
```

Comment:
Unfortunately that's Copyright law. :(

You can recreate it from scratch, but you can't derive it from this source code without it falling under GPL.